src/DynamoCore/Models/HomeWorkspaceModel.cs

```diff
@@ -131,7 +141,7 @@ public override void OnNodesModified()
             // When Dynamo is shut down, the workspace is cleared, which results
             // in Modified() being called. But, we don't want to run when we are
             // shutting down so we check that shutdown has not been requested.
-            if (DynamicRunEnabled && EngineController != null)
+            if (RunSettings.RunType != RunType.Manually && EngineController != null)
             {
                 DynamoModel.OnRequestDispatcherBeginInvoke(Run);
```

Comment:
I'm talking about DynamoModel.OnRequestDispatcherInvoke - but don't worry about it.